components/MapMarker.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <l-circle
+    v-bind="markerOptions"
+    :lat-lng="[marker.lat, marker.lng]"
+    :color="color"
+  >
+    <l-popup>
+      <v-card
+        flat
+        light
+        class="ma-0 pa-0"
+      >
+        <v-card-title>{{ marker.city }}</v-card-title>
+        <v-card-text>
+          {{ marker.description || "Lorem Ipsum" }}
+        </v-card-text>
+        <v-card-actions>
+          <v-btn
+            v-if="marker.status !== markerStatus.INPROGRESS"
+            small
+            class="text-none"
+            elevation="0"
+            color="warning"
+            @click="onClickInProgress"
+          >
+            Взято в роботу
+          </v-btn>
+          <v-btn
+            v-if="marker.status !== markerStatus.DONE"
+            small
+            class="text-none"
+            elevation="0"
+            color="success"
+            @click="onClickDone"
+          >
+            Зроблено!
+          </v-btn>
+          <v-btn
+            v-if="marker.status !== markerStatus.TODO"
+            small
+            class="text-none"
+            elevation="0"
+            color="error"
+            @click="onClickTodo"
+          >
+            Зробити
+          </v-btn>
+        </v-card-actions>
+      </v-card>
+    </l-popup>
+<!--    <l-tooltip :options="{ permanent: true, direction: 'center' }">-->
+<!--      {{ city.inquiries ?? "?" }}-->
+<!--    </l-tooltip>-->
+  </l-circle>
+</template>
+<script>
+  import { mapActions } from 'vuex'
+  import markerStatus from '~/enums/markerStatus'
+  export default {
+    props: {
+      marker: {
+        type: Object,
+        default () {
+          return {}
+        }
+      }
+    },
+    data () {
+      return {
+        markerOptions: {
+          stroke: true,
+          opacity: 0.6,
+          weight: 100,
+        },
+        markerStatus
```

Comment:
you import markerStatus and also created a variable in the data with the same name - is this done intentionally?

Reply:
yes, its the way to use imported constant markerStatus in template, 

diff button is rendered on specific marker status

Reply:
got it, resolved